app/src/test/java/by/academy/jc/klimakhovich/TestMyClass.java

```diff
@@ -0,0 +1,16 @@
+package by.academy.jc.klimakhovich;
+
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class TestMyClass {
+
+    @Test
+    void firstTestClass() {
+        MyClass classUnderTest = new MyClass();
```

Comment:
Store `classUnderTest` variable as a unit class property and initialize it in some of the two `@Before` clauses available.